spdx-utils/src/main/kotlin/SpdxUtils.kt

```diff
@@ -78,15 +80,8 @@ fun getLicenseText(id: String, handleExceptions: Boolean = false): String {
     } else {
         Pair("https://github.com/spdx/license-list-data/raw/$version/json/details/$id.json", "licenseText")
     }
-    val request = Request.Builder().get().url(url).build()
-
-    return OkHttpClientHelper.execute("utils/cache/http", request).use { response ->
-        val body = response.body()
 
-        if (response.code() != HttpURLConnection.HTTP_OK || body == null) {
-            throw IOException("Failed to download the text for license '$id' from $url.")
-        }
-
-        ObjectMapper().readTree(body.string())[key].textValue()
-    }
+    // TODO: Change this to use built-in resources instead of making HTTP requests, but for now only make it independent
+    // of "OkHttpClientHelper" from the "utils" module.
+    return ObjectMapper().readTree(URL(url).readText())[key].textValue()
```

Comment:
That function should also be in `Extensions.kt`.

Reply:
Again, why? This also is no extension function, just a temporary object creation.